servicemesh-install/topics/tutorial-prometheus.adoc

```diff
@@ -0,0 +1,67 @@
+[[prometheus_tutorial]]
+= Prometheus Tutorial
+////
+TODO
+Need a short overview of Prometheus.
+////
+
+Prerequisites:
+
+* OpenShift Container Catalog 3.10 or higher installed.
+* {ProductName} {ProductVersion} installed.
+* `Bookinfo` demonstration application installed. 
+
+== Querying Metrics
+
+. Verify that the `prometheus` service is running in your cluster. In Kubernetes environments, execute the following command:
++
+```
+$ oc get svc prometheus -n istio-system
+```
++
+You should see something like the following: 
++
+```
+NAME         CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+prometheus   10.59.241.54   <none>        9090/TCP   2m
+```
++
+. Generate network traffic by accessing the Bookinfo application:
++
+```
+curl -o /dev/null http://$GATEWAY_URL/productpage
+```
++
+. A route to access the Prometheus user interface should already exist. Query OpenShift for details of the route:
++
+```
+  $ oc get route prometheus -n istio-system
+  NAME         HOST/PORT                                  PATH      SERVICES     PORT              TERMINATION   WILDCARD
+  prometheus   prometheus-istio-system.127.0.0.1.nip.io             prometheus   http-prometheus                 None
+```
++
+. Launch a browser and navigate to  http://prometheus-istio-system.127.0.0.1.nip.io, you should see the Prometheus home screen, similar to this:
```

Comment:
This only works in 'oc cluster up' environments, it would be better to have something like

export PROMETHEUS_URL=$(oc get route -n istio-system prometheus -o jsonpath='{.spec.host}')
navigate to http://${PROMETHEUS_URL}